docs/proposals/proposals_process_proposal.MD

```diff
@@ -0,0 +1,107 @@
+
+- *New* proposals are explicitly added to the public IRC meeting agenda for each week by the meeting organizer for for acknowledgement of ongoing discussion and existence, and/or easy approval/rejection. (Either via a separate issue somewhere tracking any meeting items, or by adding a “meeting” label to the PR.)
+- Existing new, not-yet-approved proposals are reviewed weekly by meeting organizer to check for slow-moving/stalled proposals, or for flags from the proposal owner indicating that they'd like to have it addressed in the weeks meeting
+
+3: PROPOSAL APPROVED
+- Amendments needed to the proposal after IRC discussion should be made immediately.
+- The proposal status should be changed to Approved / In Progress in the document.
+- The proposal should be moved from /docs/proposals to a docs/roadmap folder (or similar).
+- The proposal PR comments should be updated with a note by the meeting organizer that the proposal has been accepted, and further commentary should be in the PRs implementing the code itself.
+- Proposals can also be PENDING (waiting on something), or DECLINED. 
+
+4: CODE IN PROGRESS
+- Approved proposals should be periodically checked for progress, especially if tied to a release and/or is noted as release blocking.
+- PRs implementing the proposal are recommended to link to the original proposal PR or document for context.
+5: CODE COMPLETE
+- Proposal document, which should be in docs/roadmap, should have their status updated to COMPLETE.
+- The release notes file for the targeted release should be updated with a small note regarding the feature or enhancement; completed proposals for community processes should have a follow-up mail sent to the mailing list providing information and links to the new process.
+- Hooray! Buy your friend a tasty beverage of their choosing.
+
+### Suggested Proposal Template Outline
+Following the .md convention, a proposal template should go in the docs/proposals repository. This is a suggested outline; the template will provide more guidance / context and will be submitted as a PR upon approval of this proposal.
+
+Please note that, in line with the above guidance that some processes will require fine-tuning over time, that the suggested template outline below, as well as the final submitted template to the docs/proposals repo has wiggle room in terms of description, and that what makes sense may vary from one proposal to another. The expectation is that people will simply do what seems right, and over time we’ll figure out what works best — but in the meantime, guidance is nice.
+
+#### TEMPLATE OUTLINE
+- Proposal Title
+- Author (w/github ID linked)
+- Date:
+
+- Status: New, Approved, Pending, Complete
+- Proposal type: Feature  / enhancement / community development process
```

Comment:
Maybe? Feature vs. enhancement could mean different things in terms of the scope of the projects, whether or not it is digging into existing code and making changes vs. adding a lot of new code. Or it could be the differentiator between "goes in release notes and marketing things, or not." We could just assume for now that features and enhancements are all just enhancements, and if we find that we need further clarity in the future, figure it out later.

I think it's useful to split out community-related processes, such as this one; they're not necessarily things that need to be in a roadmap or release notes. And may be useful to store in a separate place from roadmap / specs after approval / implementation, so folks can easily see how things evolved. But I'm not married to the idea by any means. :bride_with_veil: